source/databricks/package/balance_fixing.py

```diff
@@ -103,17 +103,16 @@ def _calculate_production(
     metadata: Metadata,
 ) -> None:
 
-    total_production_per_per_ga_and_brp_and_es = agg_steps.aggregate_production(
-        enriched_time_series, metadata
+    production_per_per_ga_and_brp_and_es = (
+        agg_steps.aggregate_production_per_ga_and_brp_and_es(
+            enriched_time_series, metadata
+        )
     )
-    total_production_per_ga_df = total_production_per_per_ga_and_brp_and_es.groupBy(
-        Colname.grid_area, Colname.time_window
-    ).agg(
-        sum(Colname.sum_quantity).alias(Colname.sum_quantity),
-        first(Colname.quality).alias(Colname.quality),
+    production_per_ga = agg_steps.aggregate_production_ga(
```

Comment:
For consistent naming either add "per" here or remove it above
```suggestion
    production_per_ga = agg_steps.aggregate_production_per_ga(
```